Added a note about activation variables

evgri243 opened the pull request:
Azure Batch Rendering uses [Software Entitlement Service for Azure Batch](https://github.com/Azure/azure-batch-software-entitlement/) that uses two special environment variables `AZ_BATCH_ACCOUNT_URL` and `AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN` to check if application was started in Azure Batch managed environment. If they are modified, the user is likely to be hit by "Software is not activated"-like message.

azurecla commented:
Hi __@BernieWhite__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@BernieWhite : Thanks for your contribution to the Azure documentation! The author, @jeevansd, has been notified to review your proposed change.

jeevansd commented:
#sign-off

Commit:
Added a note about activation variables

Azure Batch Rendering uses [Software Entitlement Service for Azure Batch](https://github.com/Azure/azure-batch-software-entitlement/) that uses two special environment variables `AZ_BATCH_ACCOUNT_URL` and `AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN` to check if application was started in Azure Batch managed environment. If they are modified, the user is likely to be hit by "Software is not activated"-like message.
